test/test_optim.py

```diff
@@ -1143,7 +1247,7 @@ def _test(self, schedulers, targets, epochs=10):
         if isinstance(schedulers, _LRScheduler):
             schedulers = [schedulers]
         for epoch in range(epochs):
-            [scheduler.step(epoch) for scheduler in schedulers]
+            [scheduler.step() for scheduler in schedulers]
```

Comment:
If `epoch` is one greater than `last_epoch` I think we should accept that in the API without complaining, and do the new behavior.